cnxauthoring/views.py

```diff
@@ -62,7 +62,12 @@ def wrapper(*args, **kwargs):
                 storage.abort()
             except storage.Error:
                 logger.exception('Storage failed to abort')
-            raise httpexceptions.HTTPServiceUnavailable()
+                try:
+                    storage.restart()
+                except storage.Error:
+                    logger.exception('Storage failed to restart')
+            finally:
+                raise httpexceptions.HTTPServiceUnavailable()
```

Comment:
Just curious if there is anyway to make this section more concise without having to result to these nested try / except statements